app/src/main/java/org/p2p/wallet/home/events/HomeScreenTokensLoader.kt

```diff
@@ -0,0 +1,51 @@
+package org.p2p.wallet.home.events
+
+import kotlinx.coroutines.flow.combine
+import org.p2p.core.token.Token
+import org.p2p.wallet.bridge.interactor.EthereumInteractor
+import org.p2p.wallet.home.ui.main.HomeInteractor
+import org.p2p.wallet.kyc.model.StrigaKycStatusBanner
+import org.p2p.wallet.striga.wallet.interactor.StrigaClaimInteractor
+import org.p2p.wallet.striga.wallet.models.StrigaClaimableToken
+import org.p2p.wallet.user.interactor.UserInteractor
+
+class HomeScreenTokensLoader(
+    private val userInteractor: UserInteractor,
+    private val ethereumInteractor: EthereumInteractor,
+    private val strigaClaimInteractor: StrigaClaimInteractor,
+    private val homeInteractor: HomeInteractor
+) : HomeScreenLoader {
+
+    override suspend fun onLoad() {
+        userInteractor.getUserTokensFlow()
+            .combine(ethereumInteractor.observeTokensFlow()) { solTokens, ethTokens ->
```

Comment:
we don't need to load eth tokens here

Reply:
it's not loading, we just collecting eth tokens, if they was loaded by someone